resources/github-markdown.css

```diff
@@ -26,6 +26,11 @@ THE SOFTWARE.
 
 */
 
+/*
+  01-12-2015
+    added media query for print;
```

Comment:
Purpose of this was that it's easier to see that it's not the original github markdown, but has some modifications.